net/hass/Makefile

```diff
@@ -0,0 +1,59 @@
+#
+# Copyright (c) 2018 Johannes Falke
+# This is free software, licensed under the GNU General Public License v3.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=hass
+PKG_VERSION:=0.1
+PKG_RELEASE:=1
+PKG_LICENSE:=GPL-3.0+
+PKG_MAINTAINER:=Johannes Falke <johannesfalke@gmail.com>
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/hass
+	SECTION:=net
+	CATEGORY:=Network
+	TITLE:=Wireless device tracker for Home Assistant
```

Comment:
ok, I missed this earlier.  If this package is _just_ the wireless device tracker, and not Home Assistant itself, then Shouldn't the package name be something like "hass-device-tracker" or something?  not just "hass" itself?

Reply:
my 2cts on this matter: I would agree